mate-frontend.md

```diff
@@ -51,6 +51,34 @@ export const ChatDeprecated: FC<Props> // ...
 export const Chat: FC<Props> // ...
 ```
 
+#### 1.3 Create new components for A/B tests
+When we run A/B tests, there's often a need to create new components. It's better to create new components instead of adding new props to existing ones. It will be easier to clean up afterward
+
+But how to handle naming in this case? Adding the `Deprecated` suffix doesn't seem right because the component doesn't become deprecated. The best solution is to add `V1`, `V2`, and so on suffixes to a new component
```

Comment:
Maybe we should add that this is V1, V2 will correspond to the variant for which it should be rendered?